src/dstack/_internal/core/models/fleets.py

```diff
@@ -172,18 +172,33 @@ class InstanceGroupParams(CoreModel):
         Optional[float],
         Field(description="The maximum instance price per hour, in dollars", gt=0.0),
     ] = None
+
+    idle_duration: Annotated[
+        Optional[Union[Literal["off"], str, int]],
+        Field(
+            description="Time to wait before terminating idle instances. Defaults to `5m` for runs and `3d` for fleets. Use `off` for unlimited duration"
```

Comment:
Not sure if `off` is better than `-1`. For me, `off` is ambiguous: does it mean “no duration” (= unlim, = -1, = don't destroy) or “no idle” (= 0, = destroy immediately after use). I personally would prefer `-1` for “never destroy”, `0` for “destroy immediately” and `> 0 ` for “destroy if idle for N days (minutes, hours, …)”

Reply:
In general I agree about `-1` being potentially better than `off`. On the other hand:
a) We already support `off` with `max_duration`
b) `-1` is a bit too technical
Perhaps, we could later support t`-1` too as a synonym for `off`. Okay?

Reply:
a) Ah, I see
b) Agree, but, as far as I understand, `-1` is already works the same way as `off`, it's just not documented. We can leave it as is for now